assignments/number-functions/code.js

```diff
@@ -0,0 +1,30 @@
+function add(x, y){
+  return x+y
+}
+function subtract(x, y){
+  return x - y
+}
+function multiply(x, y){
+  return x * y
+}
+function divide(x, y){
+  return x / y
+}
+function mod(x, y){
+  return x % y
+}
+function averageOfTwo(x, y){
+  return (x + y)/2
+}
+function averageOfThree(x, y, z){
+  return (x + y + z)/3
+}
+function distance(x, y){
+  return Math.abs(x-y)
+}
+function manhattanDistance(x1, y1, x2, y2){
+  return Math.abs(x1 - x2) + Math.abs(y1 - y2)
```

Comment:
To be even more stylish you could have written this to use the `distance` function you just defined.

```suggestion
  return distance(x1, x2) + distance(y1, y2);
```

Makes it slightly easier to read, say for someone who doesn't necessarily know what Manhattan Distance is.